test/integration/sanCirculationTest.js

```diff
@@ -0,0 +1,140 @@
+const { generateGetMetric } = require('../../generators/getMetric.js')
+const chai = require('chai')
+const expect = chai.expect
+
+const { testFieldTypes } = require('../helper.js')
+
+const {
+  testHandlesNullData,
+  assertNumberOfRecords
+} = require('../integration_helper.js')
+
+const {
+  slug,
+  from,
+  to,
+  days,
+  numberOfDays,
+  formatDate
+} = require('../setup.js')
+
+describe('circulation metric', async () => {
+  await generateGetMetric()
+  const expected = {
+    date: 'string',
+    value: 'number'
+  }
+
+  const response = san.SAN_CIRCULATION(slug, from, to)
+  const headers = response[0]
+  const results = response[1]
+  testFieldTypes(results, expected)
+  testHandlesNullData(
+    'fetchGetMetric',
+    san.SAN_CIRCULATION,
+    slug,
+    from,
+    to)
+
+  it('has proper headers', () => {
+    const expectedHeaders = ['Date', 'Value']
+    expect(headers).to.deep.equal(expectedHeaders)
+  })
+
+  it(`returns a record per every day for circulation_10y`, () => {
```

Comment:
This is a lot of code, you need some kind of helper. 
```js
  testTimeBound(
    san.SAN_CIRCULATION,
    slug,
    from,
    to,
    numberOfDays)
```

And inside the function there should be a loop on the timebounds.